cupy/cuda/driver.pxd

```diff
@@ -23,7 +23,10 @@ cdef extern from *:
     ctypedef int CUfunction_attribute 'CUfunction_attribute'
 
     ctypedef size_t(*CUoccupancyB2DSize)(int)
-
+    ctypedef struct IpcMemHandle 'cudaIpcMemHandle_t':
+        unsigned char[64] reserved
+    ctypedef struct IpcEventHandle 'cudaIpcEventHandle_t':
+        unsigned char[64] reserved
```

Comment:
1. Shouldn't these two belong to `runtime.pxd`?
2. Do you have references to the struct declarations? Oddly the official doc is empty for these two 🧐
3. Need an empty line below

Reply:
I didn't found a source with the fields, so I looked into the cuda headers myself, as the structure is passed by value to some calls instead of having a pointer.
its defined with only an array of 64 chars

Reply:
oh oh I see. This is what I found in `/usr/local/cuda/include/driver_types.h`:
```C
/**
 * CUDA IPC Handle Size
 */
#define CUDA_IPC_HANDLE_SIZE 64

/**
 * CUDA IPC event handle
 */
typedef __device_builtin__ struct __device_builtin__ cudaIpcEventHandle_st
{
    char reserved[CUDA_IPC_HANDLE_SIZE];
}cudaIpcEventHandle_t;

/**
 * CUDA IPC memory handle
 */
typedef __device_builtin__ struct __device_builtin__ cudaIpcMemHandle_st
{
    char reserved[CUDA_IPC_HANDLE_SIZE];
}cudaIpcMemHandle_t;
```
Seems like we can just use `char` instead of `unsigned char`?

Reply:
If we use char, when we convert to bytes it complains because some of the values are < 0, so I just went ahead by making it unsigned char in the definition. 
I am fine with doing explicit castings but I thought this was more readable and shouldn't affect binary compat.

Reply:
I see, I didn't know negative values could be a problem for `bytes`...